### PR TITLE
Add support for tagging newly created DigitalOcean droplets.

### PR DESCRIPTION
**Pull request by kkoch986**
### What does this PR do?
Adds support for assigning tags to newly created droplets on DigitalOcean.

### What issues does this PR fix or reference?
None

### Tests written?
No

### CODE REVIEW

File: salt/cloud/clouds/digitalocean.py [hunk 1]
```diff
@@ -298,7 +298,8 @@ def create(vm_):
         'size': get_size(vm_),
         'image': get_image(vm_),
         'region': get_location(vm_),
-        'ssh_keys': []
+        'ssh_keys': [],
+        'tags': []
```
Comment (cachedout) [inline salt/cloud/clouds/digitalocean.py]: Below, it looks like this is expected to be a string? IMHO, we should either actually make it a list and specify it that way in the configuration file, which I think is preferable, or we should set this to default to an empty string.

**Review by cachedout — CHANGES_REQUESTED** [review]

File: salt/cloud/clouds/digitalocean.py (hunk 1 above)
Comment (kkoch986) [reply salt/cloud/clouds/digitalocean.py]: yea the way i write it was just based on the `ssh_keys` param which does something similar.

```
    if ssh_key_names:
        for key in ssh_key_names.split(','):
            kwargs['ssh_keys'].append(get_keyid(key))

    key_filename = config.get_cloud_config_value(
        'ssh_key_file', vm_, __opts__, search_global=False, default=None
    )
```

We send it to DO as an array but read it from the config as a string. i can look into changing everything to an array.

Comment (cachedout) [reply salt/cloud/clouds/digitalocean.py]: Thanks, @kkoch986

Comment (cachedout) [reply salt/cloud/clouds/digitalocean.py]: Any update here @kkoch986

Comment (kkoch986) [reply salt/cloud/clouds/digitalocean.py]: just updated it sorry for the delay

**Review by rallytime — APPROVED** [review]

**Comment by rallytime** [comment]
Test failures are not related.

Thanks for updating this @kkoch986!

**Comment by kkoch986** [comment]
thanks!!

**Pull request merged**